packages/button/build.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console, import/no-extraneous-dependencies */
+import * as esbuild from 'esbuild';
+
+const isWatching = process.argv.includes('--watch');
+
+const mainBuild = esbuild
+  .build({
+    bundle: true,
+    entryPoints: ['src/index.ts'],
+    format: 'esm',
+    outdir: 'dist',
+    plugins: [
+      {
+        name: 'make-all-packages-external',
+        setup(build) {
+          const filter = /^[^./]|^\.[^./]|^\.\.[^/]/; // Must not start with "/" or "./" or "../"
+          build.onResolve({ filter }, (args) => ({
+            external: true,
+            path: args.path,
+          }));
+        },
+      },
+    ],
+    watch: isWatching
+      ? {
+          onRebuild(error, result) {
+            if (error) console.error('watch build failed:', error);
+            else console.log('watch build succeeded:', result);
+          },
+        }
+      : undefined,
+  })
+  .then(() => {
+    if (isWatching) {
+      console.log('watching...');
+    }
+  })
+  .catch(() => process.exit(1));
+
+const packageBuild = esbuild.build({
+  entryPoints: ['./node_modules/@rainbow-me/rainbowkit/styles.css'],
+  format: 'esm',
+  outdir: 'dist/rainbowkit/styles',
+  watch: isWatching
+    ? {
+        onRebuild(error, result) {
+          if (error) console.error('wallets build failed:', error);
+          else console.log('wallets build succeeded:', result);
+        },
+      }
+    : undefined,
+});
```

Comment:
Not the cleanest way, but this will ensure that we push styling into `/dist` folder so we don't rely on `node_modules` here